client/blocks/post-share/publicize-actions-list.jsx

```diff
@@ -199,7 +199,7 @@ class PublicizeActionsList extends PureComponent {
 			return <Notice
 				status="is-info"
 				showDismiss={ false }
-				text={ translate( 'Did you know you can decide exactly when Publicize shares your post? You can!' +
+				text={ translate( 'Did you know you can decide exactly when Publicize shares your post? You can! ' +
```

Comment:
@alexrford it's a custom system that we have, querying an ES index based on translate.wordpress.com. I don't think it could be useful for LGTM (which seems great, btw)